packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -348,6 +452,7 @@ class OnsetPicker extends React.Component {
 
     let ageInput = null;
     let ageUnitSelect = null;
+    let yearSelect = null;
```

Comment:
Can we declare these three variables right before each conditional setter? I think this is easier to follow. Also, nitpick, but we could leave off the null assignment - React will ignore both undefined and null children.
```jsx
let yearSelect;
if (this.state.granularity === GranularityOptions.YEAR) {
  yearSelect = this.getYearInput(intl, this.props.id);
}
```

Reply:
Resolved in 1417f94